docs/guides/admin/docs/releasenotes/analyze-mediapackage

```diff
@@ -0,0 +1,4 @@
+Changed behaviour of analyze-mediapackage
+-----------------------------------------
+
+Behaviour of analyze-mediapackage has been changed with this fix. The buggy behaviour was replacing every character that doesn't match a-z or 0-9 with underscore character. This fix only adds underscore character to the already existing flavor only to remove "/" character, just like analyze-tracks does it.
```

Comment:
```suggestion
The behaviour of the `analyze-mediapackage` workflow operation has been changed. Instead of replacing every character that doesn't match `a-z` or `0-9` with an underscore character, the operation now only replaces the `/` separating flavor and subflavor. This makes it behave identical to the `analyze-tracks` operation.
```